aws/sdk/examples/telephone-game/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "telephone-game"
+version = "0.1.0"
+authors = ["Zelda Hessler <zhessler@amazon.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+aws-config = { path = "../../build/aws-sdk/aws-config" }
+aws-sdk-polly = { package = "aws-sdk-polly", path = "../../build/aws-sdk/polly" }
+aws-sdk-s3 = { package = "aws-sdk-s3", path = "../../build/aws-sdk/s3" }
+aws-sdk-transcribe = { package = "aws-sdk-transcribe", path = "../../build/aws-sdk/transcribe" }
```

Comment:
the example looks like it uses `transcribestreaming`?

Reply:
my mistake